src/cli-access-protected-api/cli.py

```diff
@@ -19,7 +19,7 @@
 # MSAL configs
 # <ms_docref_configure_msal>
 config = {
-    # Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+    # Full directory URL, in the form of https://login.microsoftonline.com/<tenant_id>
```

Comment:
If you change it here, you need to also change it in the README.md file. They are supposed to stay in sync. So either revert this change or sync the README, either is fine.

Reply:
Hello @ckittel I've updated the README.md file as well. Thanks for this guide.

Reply:
Thanks, and good catch on the `app.py` -> `cli.py`!